stable/spinnaker/values.yaml

```diff
@@ -31,14 +31,14 @@ mail:
 
 # Images for each component
 images:
```

Comment:
Also implicit in my last comment was that since this PR hadn't been merged yet I wanted to get to the latest point release, 1.1.0, at the same time.